pkg/services/auth/auth_token.go

```diff
@@ -305,6 +306,36 @@ func (s *UserAuthTokenService) RevokeAllUserTokens(ctx context.Context, userId i
 	})
 }
 
+func (s *UserAuthTokenService) BatchRevokeAllUserTokens(ctx context.Context, userIds []int64) error {
```

Comment:
Shouldn't we add an event listener, so we could use `Dispatch`? Since right now we are bound to it?

Reply:
UserAuthTokenService is used as a service and is injected so for this func it should not be necessary.